scripts/cwl_workflows/atm-unified-eam/hrzremap_posin.cwl

```diff
@@ -37,12 +37,10 @@ arguments:
   - ncclimo
   - '-7'
   - --dfl_lvl=1
-  - --no_cll_msr
+  - --no_cell_measures
   - --no_stdin
   - -P
-  - "eam"
-  - -a
-  - sdd
+  - eam
   - --regrid_options="--rnr=0.0"
```

Comment:
I suggest changing this option from `--regrid_options="--rnr=0.0"` to `--preserve=local_mean`. It's clearer and it does the same thing in NCO 5.0.2 and later and I believe zppy is currently using NCO 5.0.3.